uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/la-admin-workpatterns.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scenarios": [
+    {
+      "label": "Contact Absence / Work Patterns",
+      "url": "civicrm/contact/view?reset=1&cid=204&selectedChild=absence#/",
```

Comment:
Without the # the tests are failing, timeboxed 15 mins for this to debug but could not find anything. We can skip this as its not that big of a issue.